queries/content/zero_width_spaces.yml

```diff
@@ -1,9 +1,9 @@
-name: "Zero-width spaces"
+name: "Zero-width spaces in a URL"
```

Comment:
the rule didn't quite seem to match what the link mentions which is more finely scoped. the original characters are actually pretty normal for HTML from my understanding